.test-infra/jenkins/job_Release_Gradle_NightlySnapshot.groovy

```diff
@@ -36,13 +36,29 @@ job('beam_Release_Gradle_NightlySnapshot') {
       'dev@beam.apache.org')
 
 
-  // Allows triggering this build against pull requests.
+  // Allows triggering this publish command against pull requests.
   common_job_properties.enablePhraseTriggeringFromPullRequest(
       delegate,
       './gradlew publish',
       'Run Gradle Publish')
 
+  // Allows triggering this build command against pull requests.
+  common_job_properties.enablePhraseTriggeringFromPullRequest(
+      delegate,
+      './gradlew clean build',
+      'Run Gradle Build')
+
   steps {
+    gradle {
+      rootBuildScriptDir(common_job_properties.checkoutDir)
+      tasks('clean')
+    }
+    gradle {
+      rootBuildScriptDir(common_job_properties.checkoutDir)
+      tasks('build')
+      common_job_properties.setGradleSwitches(delegate)
+      switches('--no-parallel')
```

Comment:
You can run a parallel build since the publishing plugins aren't enabled.

Reply:
I met some errors only occurring with 'build --no-parallel' and I'm trying to keep this build as same as what we do in release. The only concern with '--no-parallel' is it may cost more time, but since it runs nightly, maybe it's not an issue.

Reply:
Discussed and solved offline with Luke.